work3/form.py

```diff
@@ -0,0 +1,12 @@
+from wtforms import StringField,PasswordField
+from flask_wtf import FlaskForm
+from wtforms.validators import DataRequired,Email,Length,EqualTo
+
+
+class Checking(FlaskForm):
+    first = StringField("Имя_пользователя", validators=[DataRequired()])
+    second = StringField("Фамилия_пользователя", validators=[DataRequired()])
+    mail = StringField("Почта", validators=[DataRequired(), Email()])
```

Comment:
Почему бы не использовать поле  `EmailField`
```suggestion
    mail = EmailField("Почта", validators=[DataRequired(), Email()])
```